ui/src/app/settings/views/Scripts/ScriptsUpload.js

```diff
@@ -0,0 +1,189 @@
+import PropTypes from "prop-types";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Redirect } from "react-router";
+import { useDropzone } from "react-dropzone";
+import pathParse from "path-parse";
+
+import { messages } from "app/base/actions";
+import Card from "app/base/components/Card";
+import Form from "app/base/components/Form";
+import FormCardButtons from "app/base/components/FormCardButtons";
+import Row from "app/base/components/Row";
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
+
+const baseStyle = {
```

Comment:
I know we talked about this, but seeing how many styles you're adding here I think this would be best as a few classes in a CSS file. One of the benefits will be that the colours can use vanilla variables.